source/Utility/Timer.h

```diff
@@ -25,6 +25,10 @@ class Timer
 	{
 		std::cout << "\nThis took " << std::to_string(GetCurrentTime()) << " seconds.\n";
 	};
```

Comment:
Try to add newlines between function definitions :)

Reply:
Ah, of course! Forgot to add those. Will push that now!

Reply:
Added them. Could you take another look?